tldw_Server_API/app/core/Ingestion_Media_Processing/persistence.py

```
@@ -980,6 +892,92 @@ def _db_worker() -> Any:
+                    client_id=client_id,
+                    media_id=int(media_id_result),
+                    analysis_details=process_result.get("analysis_details") or {},
+                )
+                if created_visual_docs:
+                    logger.info(
+                        "Persisted %s VisualDocuments for media_id=%s (input_ref=%s)",
+                        created_visual_docs,
+                        media_id_result,
+                        original_input_ref,
+                    )
+        except Exception as visual_err:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)